cmd/terramate/cli/cli.go

```diff
@@ -637,7 +638,7 @@ func (c *cli) checkGitUncommited() bool {
 		}
 	}
 
-	cfg := c.prj.cfg.Node
+	cfg := c.prj.rootcfg.Tree().Node
```

Comment:
We seem to get the root config node multiple times, maybe we could have a method like rootNode and we cloud call:

```go
c.rootNode()
```

Or some other name.